chromoscores/snipping.py

```diff
@@ -178,64 +191,44 @@ def tad_snippet_sectors(
     return in_tad, out_tad, pile_center
 
 
-def flame_snipping_vertical(contact_map, stall_list, index, width, edge):
-    """
-    Flame snippet (vertical)
-    To extract snippets around a vertical flame due to a boundary element
-    with index "index" from a list of boundary elements "stall_list":
-    ("edge_length" is the excluded areas at the ends of the flame)
-    ------------------------------------------
-    Function flame_snippet_vertical(contact_map, stall_list, index, width,
-     edge_length):
-
-         begin function
-
-         set snippet_matrix from contact_map containing flame with a selected
-          width:
-             snippet = contact_map[
-             (stall_list[index] + edge_length):(stall_list[index + 1] -
-             edge_length),
-             (stall_list[index + 1] - width):(stall_list[index + 1] +
-             width),]
 
-    return snippet_matrix
-
-    end function
-    ------------------------------------------
+def flame_snipping_vertical(contact_map, boundary_list, index, width, edge):
+    """
+    parameters
+    ----------
+    contact_map: contact map
+    boundary_list: list of the boundary elements positions on the diagonal.
+    index: index of the boundary element in the boundary_list. This should be in the range of boundary_list.
+    width: width of the flame
+    edge: excluded areas at the ends of the flame
+    
+    returns
+    -------
+    a snippet of the contact map around the specified flame
     """
     snippet = contact_map[
-        (stall_list[index] + edge) : (stall_list[index + 1] - edge),
-        (stall_list[index + 1] - width) : (stall_list[index + 1] + width),
+        (boundary_list[index] + edge) : (boundary_list[index + 1] - edge),
+        (boundary_list[index + 1] - width) : (boundary_list[index + 1] + width),
     ]
     return snippet
 
 
-def flame_snipping_horizontal(contact_map, stall_list, index, width, edge):
+def flame_snipping_horizontal(contact_map, boundary_list, index, width, edge):
```

Comment:
I think it might be more clear if flame scores use the same on-diagonal snipper as isolation scores